internal/actor/yes_no_maybe.go

```diff
@@ -0,0 +1,10 @@
+package actor
+
+//go:generate enumerator -type YesNoMaybe -linecomment -empty
+type YesNoMaybe uint8
+
+const (
+	Yes YesNoMaybe = iota + 1
+	No
+	Maybe
+)
```

Comment:
kind of annoyed that I needed something with 3 states, and then couldn't think of a good way to differentiate from just yes/no

Reply:
I guess an alternative would be to use this around the app instead of `YesNo` but then the majority of pages wouldn't be using `Maybe` so yeah, necessary evil 🤷